### DISCUSSION TITLE
create documentation video for an existing project

### DISCUSSION THREAD

**Discussion by igorovic** [remotion-dev/remotion#76] [Ideas]
I think it will be great if we can render any website with pupeteer and script use interactions with something like [puppeterr recorder](https://github.com/puppeteer/recorder) ans use `remotion` to add an overlay with tansparency. 

This could be used to create some documentation videos for an existing project for instance. 

Maybe we can use an `iframe` in remotion to render an existing project and record a documentation overlay.

I hope I was able to clearly explain my idea. 😄 
Let me know if it's possible.

Amazing work by the way. 👍🏻

**Comment by JonnyBurger** [comment]
Nice idea! [IFrame support is now here](https://remotion.dev/docs/iframe), the challenge will be to transform puppeteer recorder scripts into something declarative.
Right now it seems to have an imperative API which doesn't work well with Remotion, since every frame has to be static and free of side-effects, to enable parallel rendering.
